### ISSUE TITLE
Custom caching

### ISSUE THREAD

**Issue by im95able** [grimfang4/SDL_FontCache#11]
Is there any way to cache the UNICODE code points I know I'll be using.
For example my app will have support for english and russian, so I would like to cache only the glyph textures belonging to those languages.

**Comment by grimfang4** [comment]
There are two ways to tell SDL_FontCache which glyphs to cache.  If you have render target support, the glyphs will be cached the first time you use a string with new codepoints in it.  Alternatively, you can cache all of your glyphs up front with FC_SetLoadingString().  Pass in a string with all of the UTF-8 codepoints you want to cache when FC_LoadFont\* is called.

**Comment by im95able** [comment]
By using a string with new codepoints, you mean calling one of the FC_Draw functions ?

**Comment by grimfang4** [comment]
Yes, the drawing functions all autocache new glyphs.  I think the others which take strings, like the metrics (e.g. FC_GetWidth), also do it.

**Comment by im95able** [comment]
Ok ok. Thank you very much for the info, and of course for the library.

**Comment by grimfang4** [comment]
You're welcome!  I haven't been running into any additional needs from the API in a while, so it might be nearly time for a real 1.0 release.
